### ISSUE TITLE
Can't fetch theme to the satisfaction of Git

### ISSUE THREAD

**Issue by Traumflug** [thirtybees/thirtybees#218]
This applies to a development installation using the Git repo directly.

Attempting to refresh submodules I get this:
```sh
$ git submodule update --init --remote --rebase
fatal: Needed a single revision
Unable to find current thirtybees/master revision in submodule path 'themes/community-theme-default'
```
Looking into `themes/community-theme-default/`, I see all the files appearing, there's a branch `thirtybees/1.0.x`and a branch `thirtybees/1.1.x`, but there is no branch `thirtybees/master`. Looking into `.gitmodules` of the main repo I see this:
```generic
[submodule "themes/community-theme-default"]
	path = themes/community-theme-default
	url = https://github.com/thirtybees/community-theme-default.git
	branch = master
	ignore = dirty
```
To me this looks like a mismatch. While I could fix this by editing `.gitmodules`, I have no idea wether the preferred solution might be to create a branch `master` in the theme repo again. Perhaps the plan is even to create different `.gitmodule` entries on each of the corresponding branches in the main repo.

**Comment by firstred** [comment]
Thanks. It should point at `1.0.x`

**Comment by Traumflug** [comment]
Problem solved with 1ed68061dbd27f08a3ca72dd63554466098a97bf. Thanks.
